test/e2e/config/caren.yaml

```diff
@@ -204,7 +204,7 @@ variables:
   # # Name of the Prism Element cluster.
   # NUTANIX_PRISM_ELEMENT_CLUSTER_NAME: ""
   # # Name of the OS image pre-uploaded in PC.
-  # NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME: ""
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME: nkp-rocky-9.4-1.29.6-20240704044125
```

Comment:
this image doesn't exist in CI PC and hence the e2e failures.

Reply:
@dkoshkin How can we get this synced between sherlock-dev and sherlock-ci?